lib/bashly/views/command/initialize.erb

```diff
@@ -2,12 +2,7 @@
 initialize() {
   version="<%= version %>"
   long_usage=''
-  set -e
-
-  if [[ "${BASH_VERSINFO:-0}" -lt 4 ]]; then
-    printf "<%= strings[:unsupported_bash_version] -%>\n"
-    exit 1
-  fi
+  set -eu -o pipefail
```

Comment:
**DANGER DANGER:** This will obviously break a lot of scripts if merged in. I stuck this here for testing but I'm not sure what the best way to handle this is. Maybe just adding this to every example's `initialize.sh`?

Reply:
We can generate this line conditionally, for example, if the environment variable `BASHLY_STRICT` is set. And then `ENV['BASHLY_STRICT'] = '1'` can be added to the spec helper.